gitopscli/git/abstract_git_util.py

```diff
@@ -53,6 +53,8 @@ def commit(self, message):
     def push(self, branch):
         try:
             self._repo.git.push("--set-upstream", "origin", branch)
+        except GitCommandError as ex:
+            raise GitOpsException(f"Error pushing branch '{branch}' to origin: {ex}") from ex
```

Comment:
I think we can reduce the log output to `GitOpsException`'s `stderr` property:
```suggestion
            raise GitOpsException(f"Error pushing branch '{branch}' to origin: {ex.stderr}") from ex
```